programming-mission/creating-effective-fatheads.md

```diff
@@ -11,14 +11,44 @@ A strong first step in planning your Fathead is to gather the set of articles yo
 
 Firstly, you can check index pages and content menus for lists of content you can incorporate. For example, the navigation menu on the left of [MDN JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/) pages has a list of language entities under "References:". Taking note of these and ensuring they are all processed by your parser will go a long way in increasing coverage.
 
-If creating a programming Fathead, you need to create a `cover/` directory containing text files that list all the topic titles and language features you need to cover. These lists will then be used as Unit tests to help you measure your Fathead's coverage. Using the example of a programming language Fathead, this list should come from a source separate to the Fathead source. For example, if using MDN you could use the [list of reserved keywords from the ECMA specification](http://ecma-international.org/ecma-262/6.0/#sec-keywords).
+If creating a programming Fathead, you need to create a `cover/` directory containing one or more text files (`.txt`) that list all the topic titles and language features you need to cover (i.e. the specific keywords ). These lists will then be used as Unit tests to help you measure your Fathead's coverage. Using the example of a programming language Fathead, this list should come from a source separate to the Fathead source. For example, if using MDN you could use the [list of reserved keywords from the ECMA specification](http://ecma-international.org/ecma-262/6.0/#sec-keywords).
```

Comment:
Perhaps clarify a bit here, e.g. instead of `These lists will then be used as Unit tests...` we could say:

    These lists are used in two ways:
      * As Unit tests to help measure your Fathead's coverage
      * To automatically create extra trigger phrases (redirects) - see below